src/main/java/org/takes/facets/fork/FkContentType.java

```diff
@@ -58,15 +56,6 @@
     /**
      * Ctor.
      * @param atype Accepted type
-     * @param response Response to return
-     */
-    public FkContentType(final String atype, final Response response) {
```

Comment:
@Happy-Neko why did you remove this constructor?
I think we should leave it as is
maybe some clients already uses this constructor